bedrock/mozorg/templates/mozorg/internet-health/open-innovation.html

```diff
@@ -127,10 +127,8 @@ <h3 class="topic-action-title">{{ _('How you can support open source') }}</h3>
               <h4>{{ _('Contribute to a project') }}</h4>
               <p>
               {% trans contribute=url('mozorg.contribute.index'),
-                       open_hatch='https://openhatch.org/'
               %}
                 Contribute to an existing open source project. Become a <a href="{{ contribute }}">Mozillian!</a>
-                Or use <a href="{{ open_hatch }}">OpenHatch</a> to match your skillset to a project.
```

Comment:
As this is changing a localized string, we're going to need to put this change behind an `l10n_has_tag` conditional (to give localizers a chance to update the translations). See [here](https://github.com/mozilla/bedrock/blob/40677449e280a7c171be006b0242cc2400d4684d/bedrock/firefox/templates/firefox/developer-quantum-whatsnew.html#L75) for an example.

Basically, we need to put the updated `{% trans %}` block (with the OpenHatch URL removed) in the `if` block, and the current `{% trans %}` block in the `else` block.

I recommend using `open-innovation-update-20171130` as the tag in the `l10n_has_tag` call.

Let me know if you have any questions or if I can help!

Reply:
Thanks, @jpetto. I'll try to get to this tomorrow.

Reply:
Passing by note: underscore is preferred to hyphen in tags ;-)

It can also be as simple as update_20171130, since its scope is the file itself (and template using it).

Reply:
The underscore is in a line which is being removed, so I won't fret about that. It may be worth adding this tip to a styleguide, though.